README.md

```diff
@@ -1,16 +1,24 @@
-# Title
+# You're a Server Communication Master
 
-## Learning Goals
+## Reflect
 
--SWBAT 1
--SWBAT 2
+Let's touch base with the reference example we started out with, our "Simple
+Liker" app. Through it, we can interact with all three of our pillars of front-
+end web programming:
 
-## Introduction
+- Manipulating the DOM
+- Recognizing JavaScript events
```

Comment:
Maybe these should be a cumulative.  This is 3/3 so all checked should confirm “ya done.”

Reply:
That's a good idea. I'll do that for the second bookend as well.